build.gradle.kts

```diff
@@ -129,6 +134,9 @@ dependencies {
         exclude("net.fabricmc", "fabric-loader")
     })
 
+    // Kilt modules
+    include(implementation(projects.kiltModlauncher)!!)
```

Comment:
If this is going to be included, it might be worth making an FMJ for the module so ModMenu can detect it and provide the correct license, name and credits.

Reply:
Sure, I was just planning to do this simply as this isn't a full module setup, but i'd be down to do more changes to make this work